examples/basic/.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  arrowParens: 'avoid',
+  bracketSameLine: true,
+  bracketSpacing: true,
+  singleQuote: true,
+  trailingComma: 'all',
+  printWidth: 100,
+};
```

Comment:
let's remove this and apply the same prettier settings as the rest of the repo :)

Reply:
done